src/pages/Admin/Admin_v2.tsx

```diff
@@ -61,12 +59,7 @@ const Admin_v2: React.FC = () => {
     <>
       <S.Wrapper className="no-venue-selected">
         <S.ViewWrapper>
-          {selectedVenue ? (
-            // @debt Venue_v2 has different structure than AnyVenue, 1 of them should be deprecated.
-            <AdminVenueView venue={selectedVenue as Venue_v2} />
-          ) : (
-            <AdminVenues venues={venues} />
-          )}
+          {selectedVenue ? <AdminVenueView /> : <AdminVenues venues={venues} />}
```

Comment:
I think we need to pass selectedVenue here so that we don't need to handle `useCurrentVenue` inside. We have this pattern in all of our Templates

Reply:
I'm glad you saw this, now that there is another new tab for the AdminVenueView, this can be removed.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1511/commits/484c4afb77a3b370f73be7ab7dbd7911b30553c0